hvplot/converter.py

```diff
@@ -911,12 +925,20 @@ def __init__(
                 'At least one resampling operation (rasterize, datashader, '
                 'downsample) must be enabled when resample_when is set.'
             )
+        if selector is not None:
+            if not _HV_GE_1_21_0:
```

Comment:
I'm unsure we should add a bokeh check here as well. It will become apparent when examining the plot.